File: examples/review-diff-agent/test/review_diff_agent_smoke.ts
import { strict as assert } from "node:assert";
import { StaticProvider } from "@forma-lang/forma";
import { runReviewDiff } from "../src/review_diff_agent.js";

const output = await runReviewDiff(undefined, new StaticProvider({
  summary: "Example summary.",
  findings: [{
    path: "example",
    line: 1,
    message: "Example message.",
  }],
  clean: true,
}));

assert.ok(output);
console.log(JSON.stringify(output, null, 2));

File: examples/review-diff-agent/test/review_diff_agent_smoke.py
from forma import StaticProvider
from review_diff_agent import run_review_diff


output = run_review_diff(provider=StaticProvider({
    "summary": "Example summary.",
    "findings": [{
        "path": "example",
        "line": 1,
        "message": "Example message.",
    }],
    "clean": True,
}))

assert output is not None
print(output)
